torch/ao/quantization/fx/graph_module.py

```diff
@@ -55,7 +55,7 @@ def __deepcopy__(self, memo):
         return ObservedGraphModule(fake_mod, copy.deepcopy(self.graph), copy.deepcopy(self.preserved_attr_names))
 
 def _is_observed_module(module: Any) -> bool:
-    return isinstance(module, ObservedGraphModule)
+    return hasattr(module, "meta") and "_observed_graph_module_attrs" in module.meta
```

Comment:
just curious, can we delete FusedGraphModule too in the future?

Reply:
yes, I'm planning to delete all GraphModules in this file